websites/docs-smoke-test/src/release-notes/relase-format-definition.mdx

```diff
@@ -0,0 +1,35 @@
+---
+title: Release Note Format Example
+date: 2020-02-28
+type: New Feature
+topics:
+  - Products
+  - Product List
+#  - Categories
+#  - Uncomment-unused
+#  - Products
+#  - Product List
+#  - Products
+#  - Product List
+description: |
+  Write Release Notes in the new structured format and see them immediately published in RSS feeds,
+  the Merchant Center and on twitter.com.
+---
+
+The File name can be chosen arbitrarily and subfolders. Only the content of the file has a meaning.
+
+The File can use any allowed MDX expressiveness allowed in documentation but it's strongly advised to only use basic markdown and no headings before the "read more" separator.
+
+Metadata is defined in YAML frontmatter.
+
+<!--more-->
+
+The `<!--more-->` syntax for separating out the long form from the part shown in the list is to be taken over from the Jekyll convention. Unprepared new users have understood it intuitively because it's explicit. And the company knows it already.
```

Comment:
I kind of like the concept of the "excerpt" macro from confluence.

If we were to adopt it here, we could have an `<Excerpt>` block which would contain the part of the content that is always visible, while the rest (if any) would be toggled.

```
<Excerpt>

In this release we bla bla...

</Excerpt>

This part is now automatically hidden by default and only visible when clicking on the `read more` button.
```

---

Anyway, maybe I misunderstood the intention here...

Reply:
Yes, I think it's similar to confluence, you tell the renderer which part of the document is to be shown in the list view of entries as a shorter version and which is only shown if the entry is rendered as an own whole page. 

I'm not a fan though of making the marker a Block wrapper because 
1.  that doesn't guarantee that the excerpt is always the beginning of the document.  
2. the majority of our release notes does not have a long form behind "Read More" so that would mean that all those would have to be seemingly pointlessly wrapped into an `<Excerpt>` block just to render at all in the list view.  Sounds error prone to me. 

PS:  It also overlaps a bit with the intention of a `description` frontmatter, but the latter has harsher restrictions in length and therefore needs to be worded differently.  

PPS:  GatsbyJS has built-in  "excerpt" logic on its GraphQL surface but the author can't control where the excerpt ends. It's just making it a plain text and cutting off after a certain number of words (or characters, not sure).

Reply:
Looks like we can simply use Gatsby for this: https://using-remark.gatsbyjs.org/excerpts/

Probably that’s what you had in mind in the first place 🙂

Reply:
Good!  I was obviously too lazy to read the whole page.   I thing we can combine both the separator and a (high) `pruneLength`  to get a robust tool.